pyaedt/edb_core/edb_data/siwave_simulation_setup_data.py

```diff
@@ -0,0 +1,897 @@
+from pyaedt.edb_core.edb_data.hfss_simulation_setup_data import EdbFrequencySweep
+from pyaedt.edb_core.general import convert_netdict_to_pydict
+from pyaedt.edb_core.general import convert_pydict_to_netdict
+from pyaedt.generic.general_methods import generate_unique_name
+from pyaedt.generic.general_methods import pyaedt_function_handler
+
+
+class SiwaveAdvancedSettings(object):
+    def __init__(self, parent):
+        self._parent = parent
+
+    @property
+    def sim_setup_info(self):
+        """Edb Internal Simulation Setup Object."""
+        return self._parent._edb_sim_setup_info
+
+    @property
+    def include_inter_plane_coupling(self):
+        """Get/Set the InterPlane Coupling.
+        The setter will enable custom settings.
```

Comment:
```suggestion
        """Interplane coupling.
        The setter enables custom settings.
```
Provide an example setting this property?